airflow/contrib/hooks/gcp_transfer_hook.py

```diff
@@ -180,14 +188,15 @@ def list_transfer_job(self, request_filter=None, **kwargs):
         if request_filter is None:
             if 'filter' in kwargs:
                 request_filter = kwargs['filter']
-                DeprecationWarning("Use 'request_filter' instead of 'filter'")
+                assert isinstance(request_filter, Dict)
+                warnings.warn("Use 'request_filter' instead of 'filter'", DeprecationWarning)
             else:
-                TypeError("list_transfer_job missing 1 required positional argument: 'request_filter'")
+                raise TypeError("list_transfer_job missing 1 required positional argument: 'request_filter'")
 
         conn = self.get_conn()
         request_filter = self._inject_project_id(request_filter, FILTER, FILTER_PROJECT_ID)
         request = conn.transferJobs().list(filter=json.dumps(request_filter))  # pylint: disable=no-member
-        jobs = []
+        jobs: List[Dict] = []
```

Comment:
Aren't variable annotation supported since Python 3.6?

Reply:
You're right. Travis did not tell me this before. We must quickly run our own CI. This will allow us to run more accurate tests.

Reply:
It will delete all annotations for variables tomorrow. :_;

Reply:
Fix: I replace it with comment annotations.